concordia/language_model/utils.py

```diff
@@ -32,6 +32,7 @@ def language_model_setup(
     model_name: str,
     api_key: str | None = None,
     disable_language_model: bool = False,
+    device: str = 'cpu'
```

Comment:
consider: could this be `Literal['cpu', 'gpu']` or somesuch to make it clearer what values are OK?

Reply:
PyTorch allows user to specify the device with custom strings. Ex: "cuda:0" for GPU 1, "cuda:1" for GPU 2, etc. so there isn't really way to wrap this nicely into a literal